src/public/modules/core/services/gtag.client.service.js

```diff
@@ -343,8 +349,8 @@ function GTag($rootScope, $window) {
    * Logs the start of a storage mode responses download.
    * @param {Object} params The response params object
    * @param {String} params.formId ID of the form
-   * @param {String} params.formTitle The title of the form 
-   * @param {String} params.userEmail The email of the user downloading
+   * @param {String} params.formTitle The title of the form
+   * @param {String} userEmail The email of the user downloading
```

Comment:
To remove this line from all relevant functions, `userEmail` is no longer a parameter